### PR TITLE
chore: add dialog for skip provision(upgrade)

### PR DESCRIPTION
**Pull request by meifans**

### CODE REVIEW

File: packages/vscode-extension/src/handlers.ts [hunk 1]
```diff
@@ -884,6 +886,38 @@ async function openMarkdownHandler() {
...
+    aadClientSecret !== "" &&
+    workspace.workspaceFolders &&
+    workspace.workspaceFolders.length > 0
+  ) {
+    try {
+      const learnMoreLink = "https://aka.ms/teamsfx-existing-aad-doc";
+      const learnMoreText = "Learn More";
+      const option = { modal: false };
+
+      const outputMsg = `Please set "AAD_APP_CLIENT_SECRET" = "${aadClientSecret}" in your system env variable. Read this wiki(${learnMoreLink}) to learn about details.`;
```
Comment (dooriya) [inline packages/vscode-extension/src/handlers.ts:907]: Please set "AAD_APP_CLIENT_SECRET" = "${aadClientSecret}" in your system environment variable. Read this wiki(${learnMoreLink}) to for more details.

**Review by dooriya — APPROVED** [review]

File: packages/vscode-extension/src/handlers.ts [hunk 2]
```diff
@@ -908,6 +910,38 @@ async function openMarkdownHandler() {
...
+  const aadClientSecretFlag = "NeedToSetAADClientSecretEnv";
+  const aadClientSecret = globalStateGet(aadClientSecretFlag, "");
+  if (
+    aadClientSecret !== "" &&
+    workspace.workspaceFolders &&
+    workspace.workspaceFolders.length > 0
+  ) {
+    try {
+      const learnMoreLink = "https://aka.ms/teamsfx-existing-aad-doc";
+      const learnMoreText = "Learn More";
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/src/handlers.ts:928]: For extension, we manage all strings in Strings.json file for potential localization.

Comment (meifans) [reply packages/vscode-extension/src/handlers.ts:928]: ok

File: packages/vscode-extension/src/handlers.ts [hunk 3]
```diff
@@ -908,6 +910,38 @@ async function openMarkdownHandler() {
...
+    workspace.workspaceFolders.length > 0
+  ) {
+    try {
+      const learnMoreLink = StringResources.vsc.upgradeToMultiEnvAndBicep.learnMoreLink;
+      const learnMoreText = StringResources.vsc.upgradeToMultiEnvAndBicep.learnMoreText;
+      const option = { modal: false };
+
+      const outputMsg = `Please set "AAD_APP_CLIENT_SECRET" = "${aadClientSecret}" in your system environment variable. Read this wiki(${learnMoreLink}) to for more details.`;
+      VsCodeLogInstance.warning(outputMsg);
+      const showMsg = `Please set "AAD_APP_CLIENT_SECRET" = "${aadClientSecret}" in your system environment variable.\nYou can also click Learn More for more details.`;
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/src/handlers.ts:933]: Same here. The principle is whether the string is visible to users, if the answer is yes, then you should extract to Strings.json file.

Comment (meifans) [reply packages/vscode-extension/src/handlers.ts:933]: ok

File: packages/vscode-extension/src/handlers.ts (hunk 2 above)
Comment (meifans) [reply packages/vscode-extension/src/handlers.ts]: fixed

File: packages/vscode-extension/src/handlers.ts (hunk 3 above)
Comment (meifans) [reply packages/vscode-extension/src/handlers.ts]: fixed

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by dooriya — APPROVED** [review]

**Pull request merged**